pkgs/applications/virtualization/ovmfvartool/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenvNoCC, fetchFromGitHub, python3 }:
+
+stdenvNoCC.mkDerivation rec {
+  name = "ovmfvartool";
```

Comment:
I'm sorry. There was an error here. It should have been `pname` instead of `name`.

Reply:
Thanks! and thanks @RaitoBezarius for the fix.